scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
As is, the following will be the most common case of a rebase:
```
12345 last_tito_commit (HEAD)   <- needs a soft reset
67890 webconsole_bump   <- needs to be dropped
11223 tito_carry_commit   <- needs a soft reset
```
I am not sure there is a sane way to deal with the above case before a `git rebase` .. 

Idea: 
1. Create e a stage branch from which we can cherry-pick in the right order.
2. `git reset --hard upstream/master`
3. Cherry-pick in the right order